README.rst

```diff
@@ -69,9 +69,9 @@ Command Completion
 
 The aws-cli package includes a very useful command completion feature.
 This feature is not automatically installed so you need to configure it manually.
-To enable tab completion for bash use the built-in command ``complete``::
 
-    $ complete -C aws_completer aws
+For bash please refer to bin/aws_bash_completer.  Add that file under /etc/bash_completion.d,
```

Comment:
I don't have strong opinion on this one, in most cases package managers take care of it for you so you don't even need to touch anything. Do you want me to keep both?